axe.d.ts

```diff
@@ -35,6 +49,10 @@ declare namespace axe {
 		resultTypes?: resultGroups[];
 	}
 	interface AxeResults {
+		toolOptions: RunOptions;
```

Comment:
Are all of these new properties _always_ in the results object? If not, we want to make them optional:

```
interface AxeResults {
  toolOptions?: RunOptions;
  testEngine?: TestEngine;
  // ...
}
```

Reply:
All reporters now have it except the raw reporter, which I thought used the `Result` interface

Reply:
OK, works for me. I just wanted to verify.